### PR TITLE
Increment Microsoft.NETCore.Platforms version

### PR DESCRIPTION
**Pull request by joshfree**
Bumped Microsoft.NETCore.Platforms version for servicing.
Included missing corert entries from running build -allconfigurations

Ran build.cmd -allconfigurations -- /p:UpdateRuntimeFiles=true
Verified package version change and resultant runtime.json changes

### CODE REVIEW

**Comment by joshfree** [comment]
Follow on PR to https://github.com/dotnet/corefx/pull/32292

File: pkg/Microsoft.NETCore.Platforms/runtime.json [hunk 1]
```diff
@@ -2685,24 +2685,62 @@
...
+        "ubuntu-x64-corert"
+      ]
+    },
     "ubuntu.18.10-x86": {
       "#import": [
         "ubuntu.18.10",
         "ubuntu-x86"
       ]
     },
+    "ubuntu.18.10-x86-corert": {
```
Comment (weshaggard) [inline pkg/Microsoft.NETCore.Platforms/runtime.json]: Are these new changes as well?

**Comment by danmosemsft** [comment]
`Windows x64 Debug Build ` failure is unrelated. @joshfree feel free to merge as this is logically part of the changed approved a couple days ago.

**Comment by joshfree** [comment]
Windows x64 Debug Build failure is a network test failure; seems to be same/similar error as reported by https://github.com/dotnet/corefx/issues/31772

[direct link to test log](https://mc.dot.net/#/user/joshfree/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Frelease~2F2.1~2F/test~2Ffunctional~2Fcli~2F/3aaa88fdc2c8cc5292cc71bb6daeae06e077ef07/workItem/System.Net.Requests.Tests/analysis/xunit/System.Net.Tests.HttpWebRequestTest~2FGetResponseAsync_GetResponseStream_ContainsHost(remoteServer:%20https:~2F~2Fcorefx-net.cloudapp.net~2FEcho.ashx))
```
System.Net.WebException : A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
---- System.Net.Http.HttpRequestException : A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
-------- System.Net.Sockets.SocketException : A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
```

**Comment by joshfree** [comment]
Marking as `servicing-approved` per shiproom rep

**Pull request merged**